### PR TITLE
Set up call center server test

### PR DESCRIPTION
**Pull request by SuaYoo**
Sets up unit testing on call center server app, with one basic passing test.

## ✋ Manual testing

1. Run `npm install` in `call-center/server`
2. Run `npm test`. Verify that you see tests pass

## 📸 Screenshots

Screen Shot 2020-10-07 at 3 14 30 PM

### CODE REVIEW

File: call-center/server/TestFactory.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import express from 'express';
+import request from 'supertest';
+import { Connection, createConnection } from 'typeorm';
+import createApp from './createApp';
+
+/* Set up and tear down server and db for tests */
```
Comment (SuaYoo) [inline call-center/server/TestFactory.ts:6]: Inspired by https://dev.to/larswaechter/unit-testing-for-apis-built-with-ts-express-js-and-typeorm-2d

File: call-center/server/createApp.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import express from 'express';
+import cors from 'cors';
+import bodyParser from 'body-parser';
+
+import calls from './routes/calls';
+import agents from './routes/agents';
+import conferences from './routes/conferences';
+
+function createApp() {
```
Comment (SuaYoo) [inline call-center/server/createApp.ts:13]: Split from `server.ts`

File: call-center/server/helpers/createConnection.ts [hunk 3]
```diff
@@ -1,12 +0,0 @@
-import 'reflect-metadata';
-import { createConnection } from 'typeorm';
-
-export default function () {
```
Comment (SuaYoo) [inline call-center/server/helpers/createConnection.ts:4]: Chore snuck in--this file wasn't being used.

File: call-center/server/server.ts [hunk 4]
```diff
@@ -1,28 +1,12 @@
...
-import bodyParser from 'body-parser';
 import { createConnection } from 'typeorm';
 
-import calls from './routes/calls';
-import agents from './routes/agents';
-import conferences from './routes/conferences';
+import createApp from './createApp';
 
 createConnection().then(function () {
-  let app = express();
```
Comment (SuaYoo) [inline call-center/server/server.ts:15]: All deletions here were moved to `createApp.ts`

**Review by DeividVeloso — APPROVED** [review]
it is looking great.

Only one suggestion/doubts:

What do you prefer put each test next to the file or create a folder test and put all test together?

**Pull request merged**